webapp/src/components/ProfilePage/ProfilePage.js

```diff
@@ -302,11 +339,11 @@ export default class ProfilePage extends React.PureComponent {
                 {isFeatureEnabled('BIDS') && (
                   <Menu.Item
                     name={PROFILE_PAGE_TABS.bids}
-                    active={this.isActive(PROFILE_PAGE_TABS.bids)}
+                    active={this.isBidActive()}
                     onClick={this.handleItemClick}
                   >
                     {t('global.bids')}
-                    {this.renderBadge(bids, PROFILE_PAGE_TABS.bids)}
+                    {this.renderBidBadge(bids)}
```

Comment:
I don't think this is bad per-se, but instead of adding a new method what about doing:

```javascript
{this.renderBadge(
  bids,
  this.isActive(PROFILE_PAGE_TABS.archivebids)
    ? PROFILE_PAGE_TABS.archivebids
    : PROFILE_PAGE_TABS.bids
)}
```